backend/models/User.js

```diff
@@ -9,16 +9,15 @@ const userSchema = new Schema({
     lastName: String,
     role: {
         type: String,
-        enum: ['Mentor', 'Mentee'],
-        // default: 'Mentee'
+        enum: ['mentor', 'mentee'],
     },
     title: {
         type: String,
-        enum: ['Product Manager', 'Designer', 'Developer'],
+        enum: ['product manager', 'designer', 'developer'],
     },
     pronouns: {
         type: String,
-        enum: ['She/Her', 'He/Him', 'They/Them', 'Xe/Xem', 'Ze/Zir'],
+        enum: ['she/her', 'he/him', 'they/them', 'xe/xem', 'ze/zir'],
```

Comment:
Does this mean you are expecting FE to send all the strings in lowerr case? Coz we dont want that. FE could send in any format just have the logic to lower call the strings received in the BE

Reply:
I think we need to be consistent on format from FE. So I expect FE always send in one format - which is lower case. Is there any reason why FE needs to send in any format?

Reply:
Yea, I agree. We have everything in FE setup based on what you had earlier, and not the lower case. However, I feel like when we are dealing with === conditions we can lower those strings in place instead of changing the schema (to save some time on the FE part)

Reply:
I'm not sure if I understand correctly. Do you mean we want to have enum values in schema as capitalisation like what we had earlier?